From 7f15c0881e1811cd7d92f201e961fcdd606e856a Mon Sep 17 00:00:00 2001
From: MilhouseVH <milhouseVH.github@nmacleod.com>
Date: Sun, 30 Jun 2019 20:06:52 +0100
Subject: [PATCH] Revert "fixed: We should always update stream details from
 player (fixes #15584)"

This reverts commit be1ed70fb4b926cbdaf60367172ac7902690f00f.
---
 xbmc/Application.cpp | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/xbmc/Application.cpp b/xbmc/Application.cpp
index 6262474d4002..e177df7cacdb 100644
--- a/xbmc/Application.cpp
+++ b/xbmc/Application.cpp
@@ -3026,8 +3026,9 @@ void CApplication::OnPlayBackStarted(const CFileItem &file)
 {
   CLog::LogF(LOGDEBUG,"CApplication::OnPlayBackStarted");
 
-  // Always update file item stream details
-  m_appPlayer.SetUpdateStreamDetails();
+  // check if VideoPlayer should set file item stream details from its current streams
+  if (file.GetProperty("get_stream_details_from_player").asBoolean())
+    m_appPlayer.SetUpdateStreamDetails();
 
   if (m_stackHelper.IsPlayingISOStack() || m_stackHelper.IsPlayingRegularStack())
     m_itemCurrentFile.reset(new CFileItem(*m_stackHelper.GetRegisteredStack(file)));
